api/rest/v1/getting_started.md

```diff
@@ -0,0 +1,161 @@
+
+```bash
+https://api.datacommons.org/v1/observations/point/variable_DCID/entity_DCID?date=YYYY&facet=XXXXXXXXXXX
+```
+
+Still confused? Each endpoint's documentation page has examples at the bottom tailored to the endpoint you're trying to use.
+
+#### Finding DCIDs
+
+Most requests require the DCID of the entity or variable you wish to query. Curious what entities and variables are available? Want to find a DCID? Take a look at our explorer tools:
+
+* [Graph Browser](https://datacommons.org/browser/) Click through the graph to see what’s available or understand its structure
+* [Place Browser](https://datacommons.org/place) Summaries of data available for entities that are geographic locations
+* [Statistical Variable Explorer](https://datacommons.org/tools/statvar) See metadata for variables that are statistical in nature
+
+#### Finding Datetimes for Observations
+
+Many endpoints allow the user to filter their results to specific dates. When querying for data at a specific date, the string passed for the date queried must match the date format (in [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601)) used by the target variable. An easy way to see what date format a variable uses is to look up your variable of interest in the  [Statistical Variable Explorer](https://datacommons.org/tools/statvar).
+
+### Bulk Retrieval
+
+Many of our APIs come in both “simple” and “bulk” versions. The simple versions of endpoints have prefix `/v1/` and are designed for handling single requests with a simplified return structure. The bulk versions of endpoints have prefix `/v1/bulk/` and are meant for querying multiple variables or entities at once, and provide richer details in the response. 
+
+#### POST requests
+
+Some bulk endpoints allow for `POST` requests. For `POST` requests, feed all parameters in JSON format as a data flag. For example, in cURL, this would look like:
+
+```bash
+POST \
+--url https://api.datacommons.org/v1/bulk/observations/point \
+--header 'content-type: application/json' \
+--data '{
+    "entities": [
+        "entity_dcid_1",
+        "entity_dcid_2",
+        ...
+    ],
+    "variables: [
+        "variable_dcid_1",
+        "variable_dcid_2",
+        ...
+    ]
+}'
+```
+
+<!--
+
+##### Pagination
+
+(Description of how pagination for bulk APIs works will go here.)
+
+### Authentication
+
+(Details about getting and setting API keys will go here.)
+-->
+
+## Troubleshooting
+
+#### Common Error Responses
+
+If your request are missing a required argument, you will receive a 400 status code and an error message like the following:
+
+```json
+{
+  "code": 3,
+  "message": "Missing required argument: stat_var",
+  "details": [
+    {
+      "@type": "type.googleapis.com/google.rpc.DebugInfo",
+      "stackEntries": [],
+      "detail": "internal"
+    }
+  ]
+}
+```
+
+If your request includes a bad argument, you will receive a 404 status code and an error message like the following:
```

Comment:
In these cases I think the APIs don't throw error while empty result is  returned for that entry.